service-admin/composer.json

```diff
@@ -13,13 +13,13 @@
         }
     },
     "require": {
-        "php": "^8.1",
+        "php": "^8.2",
```

Comment:
I would probably fix this to "8.2". That way we can't accidentally move it to 8.3 without realising.

Reply:
done